File: deepseek/1. Algoritmo Básico Ordenamiento por Burbuja/Python/burbuja.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr

# Solicitar valores al usuario
input_str = input("Ingrese números separados por espacios: ")
numbers = list(map(int, input_str.split()))
sorted_numbers = bubble_sort(numbers)
print("Lista ordenada:", sorted_numbers)

File: deepseek/1. Algoritmo Básico Ordenamiento por Burbuja/Javascript/burbuja.js
function bubbleSort(arr) {
    let n = arr.length;
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
            }
        }
    }
    return arr;
}

// Solicitar valores al usuario
let input = prompt("Ingrese números separados por comas: ");
let numbers = input.split(',').map(Number);
let sortedNumbers = bubbleSort(numbers);
console.log("Lista ordenada:", sortedNumbers);